### ISSUE TITLE
[Suggestion] Holding and Unbreaking enchants

### ISSUE THREAD

**Issue by Xiaminou** [CyberdyneCC/SimplyJetpacks-2#50]
Add the holding enchant from CoFH's mods and their version of Unbreaking which reduce the power consumed by items that use redstone flux.

**Comment by Tomson124** [comment]
I want to try not depend much on CoFH mods as their will be jetpacks for other mods in the future, too which then can not have those benefits, if they don't want to use e.g. TE. But I will have a look at it and maybe come up with something on my own.

**Comment by Xiaminou** [comment]
Sorry, I should have said that in my mind the enchants would only be for TE's jetpacks, not for all of them. Which means it would still be an optional dependency.

**Comment by Tomson124** [comment]
Okay, in that case it makes more sense :) I will probably implement it for now, but my plan for the future is to have a upgrade system for the jetpacks, which would also inlcude energy usage efficency upgrades. That also means more tweaking for the standard config values for energy usage and capacity etc.

**Comment by Xiaminou** [comment]
Sounds good, looking forward to it.
This upgrade system and the attachment system that's been mentioned in another issue will make Simply Jetpacks even more enjoyable.

**Comment by thundr** [comment]
@Tomson124 CoFH enchants are not doable currently, did you still want to work an upgrade system out?

**Comment by Tomson124** [comment]
Yeah would make sense I guess. But we have to find something that is different than other systems but nevertheless simple and not OP.
